README.md

```diff
@@ -122,1907 +122,10 @@ const { items } = await lastSucceededRunClient.dataset().listItems();
 > will be available to all runs in the future.
 
 ### Pagination
-Most methods named `list` or `listSomething` return a [<code>Promise.&lt;PaginationList&gt;</code>](#PaginationList).
+Most methods named `list` or `listSomething` return a [<code>Promise.&lt;PaginationList&gt;</code>](/api/interface/PaginatedList).
```

Comment:
hmm i guess this wont work in the readme rendered on GH? maybe better to put absolute links in ther readme, same for the api ref

Reply:
True, also, this README is not rendered anywhere else (docusaurus renders stuff from the docs folder only). 

What is the final URL of the new docs? `docs(-v2)?.apify.com`?

Reply:
> True, also, this README is not rendered anywhere else (docusaurus renders stuff from the docs folder only).

Oh, I thought we render it in the API reference as in the SDK, but I see it is not there (which is fine, as it would be duplicated).

> What is the final URL of the new docs? docs(-v2)?.apify.com?

https://docs.apify.com, and the sdk and developers subdomains will be redirected to it